images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile.yml

```diff
@@ -20,6 +20,9 @@
 - name: Configure Reporting
   include_tasks: configure_reporting.yml
 
+- name: Configure Metastore
+  include_tasks: configure_metastore.yml
```

Comment:
Nit: we should rename that task file it to something like `configure_hive_metastore.yml`.

Reply:
We may also want to bump up the section to be after the configure_storage.yml configuration.